### DISCUSSION TITLE
Github.dev is completely broken

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#5591] [Codespaces] [answered]
Files aren't loading, you can't install extensions or use any tabs. I pressed . on my fork [`console`](https://github.com/Milo123459/console) and it just simply wont load. I'm very confused.

**Comment by brettcannon** [comment, marked as answer]
Are you still having issues? I was able to loading it up with some extensions I have installed.

![image](https://user-images.githubusercontent.com/54418/135695309-0b7c6717-caac-4efd-bc8f-c891d21d3262.png)

**Reply by Milo123459** [reply]
Seems to be fixed now, like, fixed now - when I send this. I think it was a browser issue.
